base/system/userinit/livecd.c

```diff
@@ -27,13 +27,16 @@ InitLogo(PIMGINFO pImgInfo, HWND hwndDlg)
     COLORREF *pBits;
     INT line, column;
 
+    if (hDC == NULL)
```

Comment:
I moved the GetDC() at variable initiatialization level. Therefore check just after initialization and goto cleanup if hDC is invalid. Make code simpler since getDC is called at initialization level and Goto called prior to further code execution (Loadimage,...)

Ok for you ?

Reply:
I was not asking what your added code does, which is obvious.
Check again!